docs/manual/howtos/status.rst

```diff
@@ -160,12 +160,12 @@ The statuses can be accessed by the following functions on the entity:
 	|                       | SampleLostStatus               | sample_lost_status               |
 	+-----------------------+--------------------------------+----------------------------------+
 
-The following piece of code shows using statuses to make a writer wait until readers are present:
+The following code example shows statuses that make a writer wait until readers are present:
```

Comment:
"example" may imply a too large part of code, "fragment" might be a better word

Reply:
Agreed. Changed to "fragment"